nginx.conf

```diff
@@ -31,6 +31,8 @@ http {
         ssl_certificate_key /etc/ssl/private/github.com.key;
 
         location /istio/istio/releases/download/ {
+            rewrite (.*[\d.]+-)(?!linux-amd64).*(.tar.gz\/?)$ $1linux-amd64$2;
```

Comment:
Thank you for including regexr link (I used https://regex101.com/ before, seems to be comparable, but good to have another tool in the toolbox).

Anyway, few thoughts on regex:
1. `(.tar.gz\/?)` part - we probably want to match `.` explicitly vs. using "any character", so they need to be escaped.
2. We could probably avoid a negative look-ahead if that's OK to rewrite URL with the same value. `rewrite (.*[\d.]+-)(.*)(\.tar\.gz\/?)$ $1linux-amd64$3;` - basically use second capturing group for arch and replace it with `linux-amd64` (https://regexr.com/67ai9)

Reply:
The negative look ahead is to avoid a cyclic rewrite. Good catch on the dots. Probably worth a follow up or. No rush though given the limited usage of this.

Reply:
Ah, thought there was a reason for a more complicated regexp. Thank you for explaining. `.` vs. `any character` in that `.tar.gz` part is really minor obviously.

Reply:
There will only ever be a few strings being passed in here, since its coming directly from the istio codebase: 
Following this pattern: `istio-1.X.X-linux-amd64.tar.gz`, for these OS options: 
* linux-amd64
* linux-arm64
* linux-armv7
* osx
* win

You can see them live here: https://github.com/istio/istio/releases/tag/1.9.9
So I think we should be ok ?

Reply:
Yeah, I believe the current regexp is OK as is.